index.html

```diff
@@ -1,14 +1,21 @@
-<!DOCTYPE html>
```

Comment:
I noticed that this was out of date with the production `template.js` file so i synchronised them. (I was unaware that we had a different template for dev & prod 🤔 )